android/src/main/java/com/pw/droplet/braintree/Braintree.java

```diff
@@ -6,7 +6,7 @@
 
 import androidx.fragment.app.FragmentActivity;
 
-import com.braintreepayments.api.BraintreeFragment;
+import com.braintreepayments.api.Client;
```

Comment:
decoupled BraintreeFragment incorrectly implemented. Each service needs to have its own Client. eg. GooglePayClient. Need to make change